openage/convert/dataformat/members.py

```diff
@@ -671,8 +671,8 @@ def get_snippets(self, file_name, format_):
 
             # function to recursively read the referenced files
             txt.extend((
-                "int %s::recurse(openage::util::Dir basedir) {\n" % (self.type_name),
-                "\tthis->index_file.read(basedir); //read ref-file entries\n",
+                "int %s::recurse(openage::util::Dir basedir, openage::util::csv_file_map_t *file_map) {\n" % (self.type_name),
+                "\tthis->index_file.read(basedir, file_map); //read ref-file entries\n",
```

Comment:
with the asset manager addition, one would recurse with a asset manager pointer here